app/src/main/java/io/homeassistant/companion/android/notifications/MessagingService.kt

```diff
@@ -50,11 +51,23 @@ class MessagingService : FirebaseMessagingService() {
         }
 
         remoteMessage.notification?.let {
-            Log.d(TAG, "Message Notification: ${it.title} -> ${it.body}")
-            sendNotification(it.title, it.body!!)
+            if (it.title == "request_location_update") {
```

Comment:
I just checked what the iOS app is doing, this should be message, not title.  That way we try and stay inline.